champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/DefensiveAura.java

```diff
@@ -104,10 +108,47 @@ public void activate(Player player, int level) {
                 if (targetMaxHealth != null) {
                     UtilPlayer.health(target, 4d * healthBoostStrength);
                     UtilMessage.simpleMessage(target, getClassType().getPrefix(), "<yellow>%s</yellow> cast <green>%s</green> on you!", player.getName(), getName());
-                    target.playSound(target, Sound.ENTITY_VILLAGER_WORK_CLERIC, 1f, 0.8f);
+                    target.playSound(target, Sound.ENTITY_VILLAGER_WORK_CLERIC, 1f, 1.1f);
+                    target.spawnParticle(Particle.HEART, target.getLocation().add(new Vector(0, 1, 0)), 6, 0.5, 0.5, 0.5);
                 }
             }
         }
+
+        new BukkitRunnable() {
+            final double rIncrement = 1.0;
+            double r = rIncrement;
+            int count = 0;
+            final Location center = player.getLocation();
+            final int colorIncrement = (int)(255 / getRadius(level) * rIncrement);
+
+            @Override
+            public void run() {
+                for (int degree = 0; degree < 360; degree += 10) {
+                    double addX = r * Math.sin(Math.toRadians(degree));
+                    double addY = 0.2;
+                    double addZ = r * Math.cos(Math.toRadians(degree));
+                    Location newLocation = new Location(center.getWorld(), center.getX() + addX, center.getY() + addY, center.getZ() + addZ);
+                    if (r < getRadius(level)) {
+                        Particle.DUST.builder()
+                                .data(new Particle.DustOptions(org.bukkit.Color.fromRGB(255, Math.max(255 - colorIncrement * count, 0), Math.max(255 - colorIncrement * count, 0)), 1.5f))
+                                .location(newLocation)
+                                .receivers(48)
+                                .spawn();
+                    } else {
+                        Particle.HEART.builder()
+                                .location(newLocation)
+                                .receivers(48)
+                                .spawn();
```

Comment:
Recommend getting a list of receivers and storing it before the loop and just passing in the collection